docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -156,13 +156,20 @@
                         others' checkpoints, and perhaps interfere with each other in other ways.
                     </td>
                 </tr>
-                <tr>
                     <td class="property" id="job-coordinator-system">job.coordinator.system</td>
                     <td class="default"></td>
                     <td class="description">
                         <strong>Required:</strong> The <span class="system">system-name</span> to use for creating and maintaining the <a href="../container/coordinator-stream.html">Coordinator Stream</a>.
                     </td>
                 </tr>
+                <tr>
+                    <td class="property" id="job-default-system">job.default.system</td>
```

Comment:
Let's use job.default.system for yet another reason: job.io.system also implicitly forces input and output systems into one, which is different than saying: if you don't specify, input/output system would be in the default.

I have another question related to scoping of the configuration: is this at job-level or at the whole application-level? Right now, I saw three levels of configuration scopes: application ==> job ==> task. Curious whether this "default IO system" should be default for the whole application or a specific job?